_posts/blog/2016-11-16-simpler-scripts-and-config.md

```diff
@@ -0,0 +1,163 @@
+
+### Better usage
+
+Before 2.0.0, the 'accumulo' command had a limited usage:
+
+```
+$ ./accumulo-1.8.0/bin/accumulo
+accumulo admin | check-server-config | classpath | create-token | gc | help | info | init | jar <jar> [<main class>] args |
+  login-info | master | minicluster | monitor | proxy | rfile-info | shell | tracer | tserver | version | zookeeper | <accumulo class> args
+```
+
+For 2.0.0, all 'accumulo' commands were given a short description and organized into the groups.
+
+```
+$ ./accumulo-2.0.0/bin/accumulo help
+
+Usage: accumulo <command> (<argument> ...)
+
+Core Commands:
+  create-config                  Create Accumulo configuration
+  build-native                   Build Accumulo native libraries
+  init                           Initializes Accumulo
+  shell                          Run Accumulo shell
+  classpath                      Print Accumulo classpath
+  version                        Print Accumulo version
+  admin                          Execute administrative commands
+  info                           Print Accumulo cluster info
+  help                           Prints usage
+  jar <jar> [<main class>] args  Run Java <main class> in <jar> using Accumulo classpath
+  <main class> args              Run Java <main class> located on Accumulo classpath
+
+Process Commands:
+  gc                             Start Accumulo garbage collector
+  master                         Start Accumulo master
+  monitor                        Start Accumulo monitor
+  minicluster                    Start Accumulo minicluster
+  proxy                          Start Accumulo proxy
+  tserver                        Start Accumulo tablet server
+  tracer                         Start Accumulo tracer
+  zookeeper                      Start Apache Zookeeper instance
+
+Advanced Commands:
+  check-server-config            Checks server config
+  create-token                   Creates authentication token
+  login-info                     Print Accumulo login info
+  rfile-info                     Print rfile info
+```
+
+The new 'accumulo-service' and 'accumulo-cluster' commands also have informative usage.
+
+```
+$ ./accumulo-2.0.0/bin/accumulo-service 
+
+Usage: accumulo-service <service> <command>
+
+Services:
+  gc          Accumulo garbage collector
+  monitor     Accumulo monitor
+  master      Accumulo master
+  tserver     Accumulo tserver
+  tracer      Accumulo tracter
+
+Commands:
+  start       Start service
+  stop        Stop service
+  kill        Kill service
+
+$ ./accumulo-2.0.0/bin/accumulo-cluster 
+
+Usage: accumulo-cluster <command> (<argument> ...)
+
+Commands:
+  create-config   Creates cluster config
+  start           Starts Accumulo cluster
+  stop            Stops Accumulo cluster
+```
+
+[ACCUMULO-4490]: https://issues.apache.org/jira/browse/ACCUMULO-4490
+[INSTALL.md]: https://github.com/apache/accumulo/blob/master/INSTALL.md
```

Comment:
Note: Link will need to be updated upon release, to point to the tag or commit, so it doesn't get stale.